### PR TITLE
9674: Session Copy Update: Add Trial Session Assignment Info to Session Copy

### PR DESCRIPTION
**Pull request by shumway-tm**
Link to [Zenhub Story.](https://app.zenhub.com/workspaces/flexionef-cms-5bbe4bed4b5806bc2bec65d3/issues/flexion/ef-cms/9674)

---

### Completed first top level task and associated subtasks.

B51CB2A8-F628-4EE7-B5D1-E39FD08A2651_4_5005_c

---
### Updated Session Copy Page with Assignments Card

image (1)

### CODE REVIEW

File: web-client/src/views/TrialSessionWorkingCopy/SessionAssignments.jsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+          <h3 className="underlined">Assignments</h3>
+          <div className="grid-container padding-x-0">
+            <div className="grid-row grid-gap">
+              <div className="grid-col-6">
+                <p className="label">Judge</p>
+                <p className="margin-bottom-0">
+                  {formattedTrialSessionDetails.formattedJudge}
+                </p>
+                <p>
+                  {formattedTrialSessionDetails.chambersPhoneNumber
```
Comment (rachelschneiderman) [inline web-client/src/views/TrialSessionWorkingCopy/SessionAssignments.jsx]: Nit: Could this logic be brought into the helper?

File: web-client/src/views/TrialSessionWorkingCopy/SessionAssignments.jsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
+import { connect } from '@cerebral/react';
```
Comment (rachelschneiderman) [inline web-client/src/views/TrialSessionWorkingCopy/SessionAssignments.jsx:1]: Nice refactor! This is great!

**Review by rachelschneiderman — APPROVED** [review]
Was package-lock committed on purpose?

File: web-client/src/views/TrialSessionWorkingCopy/SessionNotes.jsx [hunk 3]
```diff
@@ -19,57 +19,51 @@ export const SessionNotes = connect(
...
-                            }}
-                          >
-                            Delete Note
-                          </Button>
-                        </div>
-                      </div>
-                    </If>
+          <div className="card">
+            <div className="content-wrapper">
+              <If not bind="trialSessionWorkingCopy.sessionNotes">
```
Comment (TomElliottFlexion) [inline web-client/src/views/TrialSessionWorkingCopy/SessionNotes.jsx:24]: First time I've seen this component used, cool

**Review by TomElliottFlexion — APPROVED** [review]

File: web-client/src/views/TrialSessionWorkingCopy/SessionAssignments.jsx (hunk 2 above)
Comment (shumway-tm) [reply web-client/src/views/TrialSessionWorkingCopy/SessionAssignments.jsx:1]: Thank you!

File: web-client/src/views/TrialSessionWorkingCopy/SessionNotes.jsx (hunk 3 above)
Comment (shumway-tm) [reply web-client/src/views/TrialSessionWorkingCopy/SessionNotes.jsx:24]: 😎

**Comment by shumway-tm** [comment]
> Was package-lock committed on purpose?

@rachelschneiderman Good catch! Definitely not done intentionally. I have reverted it.

File: web-client/src/views/TrialSessionWorkingCopy/SessionAssignments.jsx (hunk 1 above)
Comment (shumway-tm) [reply web-client/src/views/TrialSessionWorkingCopy/SessionAssignments.jsx]: We have extracted the logic into the helper! Thank you for the suggestion!

**Pull request merged**